### DISCUSSION TITLE
Supporting Sliver financially?

### DISCUSSION THREAD

**Discussion by Nariod** [BishopFox/sliver#753] [Q&A] [answered]
Hello there,

Well, everything's in the title. For teams not having Golang skills but interested in supporting Sliver, is supporting Sliver in a financial way something the Sliver team is looking for?

Regards,

**Comment by moloch--** [comment, marked as answer]
Thank you for the consideration, however we're not looking for any financial support at this time. If you'd like to show appreciation for the project consider a donation to the EFF in our name.
